README.md

```diff
@@ -954,42 +954,72 @@ Used for EV chargers.
 
 #### Interfaces
 
-Type | Interface                      | Value type | Properties              | Description
------|--------------------------------|------------|-------------------------|------------
-in   | cmd.charge.start               | null       |                         | Start charging (allow cars to charge) OCPP: Remote Start Transaction
-in   | cmd.charge.stop                | null       |                         | Stop charging (stop cars from charging) OCPP: Remote Stop Transaction
+Type | Interface                       | Value type | Properties              | Description
+-----|---------------------------------|----------|-------------------------|------------
+in   | cmd.charge.start                | null     |                         | Start charging (allow cars to charge) OCPP: Remote Start Transaction
```

Comment:
Let's give @bjhandeland no reasons to play a grammar tyrant here! :-) Add periods at the end of each sentence in the description column.

Reply:
Done.